.github/workflows/maven-ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Maven CI
+
+on: [push, pull_request]
```

Comment:
Unlike OTP-RR, we DO want to keep the push event for deploying and testing branch code.